R/util.R

```diff
@@ -471,3 +471,18 @@ unlist_times <- function(x) {
 empty_time <- function() {
   Sys.time()[-1]
 }
+
+
+maybe_unlink <- function(x, recursive = FALSE, dry_run = FALSE) {
+  if (!dry_run) {
+    unlink(x, recursive)
+    return()
+  }
+  recurse <- if (recursive) " recursively."
+  cli::cli_alert_info("Dry_run - would have deleted:")
+  ul <- cli::cli_ul()
+  for (file in x) {
+    cli::cli_li("{file}{recurse}")
+  }
+  cli::cli_end(ul)
```

Comment:
```suggestion
  cli::cli_bullets(paste0(file, recurse))
```

(this formats very slightly differently out the box but you can tame it)

Reply:
Done - basically
    files <- paste0(x, recurse)
    names(files) <- rep("*", length(files))
    cli::cli_bullets(files)